execute_pipeline.py

```diff
@@ -137,5 +188,31 @@ def _var_replace(in_str, repl_vars):
   return new_str
 
 
+def _load_local_repo(private_repo_root, **pipeline_kwargs):
+  files_dict = {}
+  for root, dirs, files in os.walk(private_repo_root):
+      for fname in files:
+        path = os.path.join(root, fname)
+        rel_path = os.path.relpath(path, private_repo_root)
```

Comment:
This is quite nitpicking, but in case the local environment is Windows, `rel_path` may contain '\' for the path separator and that would confuse the remote system pretty much. I guess it's safer to invoke `os.path.split()` and use the list of path components instead of path string.